nml/version_info.py

```diff
@@ -135,15 +135,23 @@ def get_cli_version():
     return result[0:-1] #strip trailing newline
 
 def get_and_write_version():
+    try:
+        from nml import __version__
+        version = __version__.version
+        if __version__.release:
+            return version
+    except ImportError:
+        pass
```

Comment:
It does not seem so nice to import __version__ here, but also in `get_nml_version()`. Couldn't all of this be handled in `ge_nml_version()`? Maybe something like this (untested, still needs comments):

```
def get_nml_version():
    __version__ = None
    try:
        from nml import __version__
		if __version__.release:
			return (__version__.version, __version__.release)
    except ImportError:
		pass
    version = get_git_version()
    if version:
        return (version, False)

	if __version__:
		return (__version__.version, __version__.release)
    return ("unknown", False)
```

Then `get_and_write_version` could check the second part of the result to see whether a release version was loaded and only write when it was True.